plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginTemporaryTester.java

```diff
@@ -54,8 +54,8 @@ private File getCredentialFile() throws IOException {
     }
 
     Map<String, String> credentialMap = new HashMap<>();
-    credentialMap.put(CLIENT_ID_LABEL, GoogleLoginService.OAUTH_CLIENT_ID);
-    credentialMap.put(CLIENT_SECRET_LABEL, GoogleLoginService.OAUTH_CLIENT_SECRET);
+    credentialMap.put(CLIENT_ID_LABEL, Constants.OAUTH_CLIENT_ID);
```

Comment:
It's useful to support retrieving these values from properties or the environment for development.  See [GPE](https://github.com/GoogleCloudPlatform/google-plugin-for-eclipse/blob/master/plugins/com.google.gdt.eclipse.login/src/com/google/gdt/eclipse/login/GoogleLogin.java#L80) for an example